src/Controller/Admin/EventController.php

```diff
@@ -140,11 +159,41 @@ protected function mapDataToEntity(array $data, Event $entity): void
         $imageId = $data['image']['id'] ?? null;
 
         $entity->setName($data['name']);
+        $entity->setRoutePath($data['routePath']);
         $entity->setImage($imageId ? $this->mediaManager->getEntityById($imageId) : null);
         $entity->setStartDate($data['startDate'] ? new \DateTimeImmutable($data['startDate']) : null);
         $entity->setEndDate($data['endDate'] ? new \DateTimeImmutable($data['endDate']) : null);
     }
 
+    protected function updateRoutesForEntity(Event $entity): void
+    {
+        // create route for all locales of the application because event entity is not localized
+        foreach ($this->webspaceManager->getAllLocales() as $locale) {
```

Comment:
What's the reason of going through all locales, when updating a route for specific locale? This usually would throw an error with NULL for the rest of the locales that are not currently edited/created